src/App.jsx

```diff
@@ -1,35 +1,28 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useEffect, useState } from "react";
+import Nav from "./Components/Nav";
+import { Routes, Route } from "react-router-dom";
+import axios from "axios";
+import ListOfArticles from "./Components/ListOfArticles";
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [articles, setArticles] = useState([]);
```

Comment:
Does this state need to be in the App component, or could it be initialised in `ListOfArticles`?

Reply:
thank you for your feedback Katherine. sorry just a quick question. I have the whole list of articles showing on the homepage but I was also planning to add a 'my articles' page in the Nav that would need access to articles to show the user all of their own articles. would that be a good reason to put the state in the app?